### PR TITLE
Dask

### PR DESCRIPTION
**Pull request by romulogoncalves**

### CODE REVIEW

**Comment by sverhoeven** [comment]
I am missing dask stuff in documentation and host groups template so I added it

File: vars/common_vars.yml.template [hunk 1]
```diff
@@ -42,3 +42,5 @@ system_packages:
 create_swap_space: false
 
 emma_modules_dir: "/data/local/{{ lookup('env', 'CLUSTER_NAME') }}/modules"
+
+public_network_interface: "ansible_enp0s8"
```
Comment (sverhoeven) [inline vars/common_vars.yml.template]: Why was this moved? It is still only used in the cassandra role.

Comment (romulogoncalves) [reply vars/common_vars.yml.template]: I was using it in Dask, but then I reverted it. Somehow I forgot to revert this one.
I just reverted the commit.

**Comment by sverhoeven** [comment]
@romulogoncalves are you OK with my changes?

**Comment by sverhoeven** [comment]
You are OK with it.

**Comment by romulogoncalves** [comment]
Yes
